src/main/java/io/cdap/e2e/pages/actions/CdfPipelineRunAction.java

```diff
@@ -93,6 +95,13 @@ public static Boolean isProvisioning() {
     return validatePipelineExpectedStatus(CdfPipelineRunLocators.provisioningStatus);
   }
 
+  /**
+   * Wait till the Pipeline's status changes to Running
+   */
+  public static void waitForPipelineToBeInRunningStatus() {
```

Comment:
Does it make sense to make this method generic and accept status and timeout as params?
It takes some time for pipeline to transition in running state, so it would be better to keep timeout configurable.

Reply:
Updated the method to expect Status and Timeout.